frontend/tests/unit/utils.shootEditorCompletions.spec.js

```diff
@@ -106,10 +71,59 @@ const shootCompletions = {
     }
   }
 }
-const shootEditorCompletions = new ShootEditorCompletions(shootCompletions, editor.options.indentUnit)
 
 describe('utils', function () {
   describe('shootEditorCompletions', function () {
+    let editor
+    let setEditorContentAndCursor
+    let setEditorCursor
+    let shootEditorCompletions
+
+    before(function () {
+      // cannot stub via sandbo as Jest does not implement the function and
+      // non existant functions cannot be stubbed using a sinon sandbox
+      document.body.createTextRange = function () {
```

Comment:
stub with sandbox

Reply:
not possible, see https://github.com/sinonjs/sinon/pull/1557
Or how would you do this?